### PR TITLE
This creates the correct result for valid csv files

### PR DESCRIPTION
**Pull request by aahladmanas**
Two features added/improved : 

1. the main.validate_hpo_files function now loads the cdm files from cloud into bigquery tables.
2. creates result csv with (found, parsed, loaded) columns filled in reflecting a successful load.

### CODE REVIEW

File: data_steward/validation/main.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env python
 import logging
 from flask import Flask
+import os
```
Comment (mark-velez) [inline data_steward/validation/main.py]: I think we can remove this import.

File: data_steward/validation/main.py [hunk 2]
```diff
@@ -49,11 +53,57 @@ def validate_hpo_files(hpo_id):
...
     warnings = [
         (unknown_file['name'], UNKNOWN_FILE) for unknown_file in unknown_files
     ]
 
+    # the following loop is used instead of python list-comprehension but the
+    # problem is tha cdm_file order gets garbled because of python dict
+    result = []
+    for cdm_file in common.CDM_FILES:
+        status = cdm_file_status_map[cdm_file]
+        result = result + [(cdm_file,status['found'],status['parsed'],status['loaded'])]
```
Comment (mark-velez) [inline data_steward/validation/main.py]: Please use list's `append`.

File: data_steward/validation/main.py [hunk 3]
```diff
@@ -49,11 +53,57 @@ def validate_hpo_files(hpo_id):
         (cdm_file, 1 if cdm_file in map(lambda f: f['name'], found_cdm_files) else 0) for cdm_file in common.CDM_FILES
     ]
 
+    # (filename, found, parsed, loaded) for each expected cdm file
+    cdm_file_status_map = {}
+    for cdm_file in common.CDM_FILES:
+        if cdm_file in map(lambda f: f['name'], found_cdm_files) :
```
[the remark this answers is not in the document]
Comment (aahladmanas) [reply data_steward/validation/main.py]: I am removing result deadcode. line 59 there is not an explicit check. I use it to create the corresponding found/parsed/loaded result.

[the remark this answers is not in the document]
Comment (aahladmanas) [reply data_steward/validation/main.py]: wait I see what you are saying.

File: data_steward/validation/main.py [hunk 4]
```diff
@@ -49,11 +53,57 @@ def validate_hpo_files(hpo_id):
...
+            # This is where the parse_check would go.
+            # don't run the rest of this is csv not parseable
+            
+            # run a bq_load jb
+            result = bq_utils.load_table_from_bucket('chci',
```
[the remark this answers is not in the document]
Comment (aahladmanas) [reply data_steward/validation/main.py]: Yep.

File: data_steward/validation/main.py [hunk 5]
```diff
@@ -49,11 +53,57 @@ def validate_hpo_files(hpo_id):
...
+                                                     cdm_file.split('.')[0])
+
+            load_job_id = result['jobReference']['jobId']
+            job_running = True
+
+            wait_count = 100
```
[the remark this answers is not in the document]
Comment (aahladmanas) [reply data_steward/validation/main.py]: Done. Waiting 10 seconds.

File: data_steward/validation/main.py (hunk 2 above)
Comment (aahladmanas) [reply data_steward/validation/main.py]: Yep. Did that.

File: data_steward/test/unit_test/validation_test.py [hunk 6]
```diff
@@ -71,7 +71,7 @@ def test_validate_missing_files_output(self, mock_check_cron):
     @mock.patch('api_util.check_cron')
     def test_all_files_unparseable_output(self, mock_check_cron):
         for cdm_table in common.CDM_FILES:
-            self._write_cloud_str(self.hpo_bucket, cdm_table, ".")
+            self._write_cloud_str(self.hpo_bucket, cdm_table, ".\n .")
```
[the remark this answers is not in the document]
Comment (aahladmanas) [reply data_steward/test/unit_test/validation_test.py]: yeah. bigquery in the current setting is ignoring the first line.

**Pull request closed without merging**